docker-for-windows/edge-release-notes.md

```diff
@@ -64,6 +92,7 @@ This release contains a Kubernetes upgrade. Note that your local Kubernetes clus
 
 - Improved the navigation in **Settings** and **Troubleshoot** UI.
 - Fixed a bug that prevented users from accessing WSL 2 Tech Preview. Fixes [docker/for-win#4734](https://github.com/docker/for-win/issues/4734).
+- Docker Machine is not included in the Docker Desktop installer anymore. You can download it separately from the [Docker Machine releases](https://github.com/docker/machine/releases) page.
```

Comment:
Suggest: Docker Machine is no longer included in the Docker Desktop installer.

Reply:
Fixed.